### PR TITLE
OSG projects for Christina's 2021 Virtual School mentees

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: projects/CSM_BeEST.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name: CSM_BeEST
```
Comment (matyasselmeci) [inline projects/CSM_BeEST.yaml:1]: In the past you've used "Mines_" as the prefix for Colorado School of Mines projects.

**Comment by matyasselmeci** [comment]
Odd that osg-bot didn't say anything about "University of Edinburgh" and "University of Sheffield" but they're both legit new organizations so that's fine; I'll add them to the mappings file after I merge this.

Is it too late to change `CSM_BeEST` to `Mines_BeEST`?

**Comment by ChristinaLK** [comment]
CSM_BeAST is the name of the project on the OSG account management side, so I think it has to match, right?

**Comment by matyasselmeci** [comment]
I don't know; you'll have to ask them. (Would that be Mats?)

**Review by matyasselmeci — APPROVED** [review]

**Pull request merged**